<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="videoFrame" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>870</width>
      <height>410</height>
     </rect>
    </property>
    <widget class="QLabel" name="showBlock">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>870</width>
       <height>410</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="selectVideo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择视频</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectDet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>选择检测器</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTracking">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始跟踪</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="videoPath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>621</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="videoSlider">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>791</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="detectorPath">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>621</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="vidControl">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>430</y>
      <width>61</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string>播放</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startDetecting">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>430</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始检测</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoTimer">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>410</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00/00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
